tests/cupy_tests/core_tests/test_core.py

```diff
@@ -5,23 +5,6 @@
 from cupy import testing
 
 
-class TestGetStridesForNocopyReshape(unittest.TestCase):
```

Comment:
Btw, this test has been removed because `_get_strides_for_nocopy_reshape` is no longer exposed now.

Reply:
OK.